ms2query/create_sqlite_database.py

```diff
@@ -31,37 +34,99 @@ def make_sqlfile_wrapper(sqlite_file_name: str,
         Name of sqlite_file that should be created, if it already exists the
         tables are added. If the tables in this sqlite file already exist, they
         will be overwritten.
-    npy_file_location:
-        The file location of an .npy file. This file is expected to contain the
-        tanimoto scores and have an equal amount of rows and columns equal to
-        the number of inchikeys in csv_file_with_inchikey_order.
-    pickled_file_name:
-        This file is expected to contain a list of matchms.Spectrum.
-    csv_file_with_inchikey_order:
-        This csv file is expected to contain inchikeys on the second column
-        starting from the second row. These inchikeys should be ordered in the
-        same way the tanimoto scores are ordered in the npy_file_location.
+    tanimoto_scores_pickled_dataframe_file:
+        A pickled file with tanimoto scores. The column names and indexes are
+        14inchikeys.
```

Comment:
Check consistent use: I believe we elsewhere (at least in matchms/ms2deepscore...) used inchikey14

Reply:
Yes, this was even worse, since I was not even consistent in my own files, most of the times I just named it inchikey (when it actually almost always is an inchikey14), I changed this in all files.